web/e2e/page-objects/jobs/jobs.ts

```diff
@@ -148,6 +156,64 @@ export class JobsPage extends AppPage {
   jobOutputCloseButton() {
     return element(by.css('app-job-output mdl-button'));
   }
+
+
+  resetFiltersButton() {
```

Comment:
some of these page objects like filtering are already exist on manageJobs.ts. Can you just delete jobs.ts and use manageJobs.ts, so we are consistent with manageFlows structure. Also, you can re-use the page objects on manageJobs.ts, we use these page objects on scenario tests. If you make some changes, make sure that it's not breaking the scenarios tests

Reply:
I moved methods to manageJobs.ts page object, will review and remove unused classes later.